ext/ext.go

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
If the file contains only package level documentation then please rename it to `doc.go`

Example from the kubernetes https://github.com/kubernetes/kubernetes/blob/master/pkg/controlplane/doc.go

Reply:
updated